TST

```
FAILED test/test_transformer.py::test_transform_bounds__ignore_inf[ESRI:102036-input_bounds0-expected_bounds0] - AssertionError: 
Arrays are not almost equal to 0 decimals

Mismatched elements: 3 / 4 (75%)
Max absolute difference among violations: 50132027.16623252
Max relative difference among violations: inf
 ACTUAL: array([-5.e+07, -1.e+08,  5.e+07,  5.e+07])
 DESIRED: array([         0, -116576599,          0,          0])
FAILED test/test_transformer.py::test_transform_bounds__ignore_inf[ESRI:54026-input_bounds1-expected_bounds1] - AssertionError: 
Arrays are not almost equal to 0 decimals

Mismatched elements: 2 / 4 (50%)
Max absolute difference among violations: 88721834.46860172
Max relative difference among violations: inf
 ACTUAL: array([-9.e+07, -2.e+08,  9.e+07,  2.e+08])
 DESIRED: array([         0, -179545824,          0,  179545824])
```

--- a/docs/history.rst
+++ b/docs/history.rst
@@ -9,6 +9,7 @@ Change Log
 - DEP: Add win_arm64 wheels
 - DEP: Add Python 3.13 free-threading wheels
 - DEP: Add Python 3.14 and 3.14 free-threading wheels
+- TST: Remove irrelevant test with modern PROJ versions
 
 3.7.1
 ------

--- a/test/test_transformer.py
+++ b/test/test_transformer.py
@@ -1310,27 +1310,6 @@ def test_transform_bounds__beyond_global_bounds():
     )
 
 
-@pytest.mark.parametrize(
-    "input_crs,input_bounds,expected_bounds",
-    [
-        (
-            "ESRI:102036",
-            (-180.0, -90.0, 180.0, 1.3),
-            (0, -116576599, 0, 0),
-        ),
-        ("ESRI:54026", (-180.0, -90.0, 180.0, 90.0), (0, -179545824, 0, 179545824)),
-    ],
-)
-def test_transform_bounds__ignore_inf(input_crs, input_bounds, expected_bounds):
-    crs = CRS(input_crs)
-    transformer = Transformer.from_crs(crs.geodetic_crs, crs, always_xy=True)
-    assert_almost_equal(
-        transformer.transform_bounds(*input_bounds),
-        expected_bounds,
-        decimal=0,
-    )
-
-
 def test_transform_bounds__ignore_inf_geographic():
     crs_wkt = (
         'PROJCS["Interrupted_Goode_Homolosine",'

